Resources/Locale/en-US/game-ticking/game-presets/preset-traitor.ftl

```diff
@@ -36,6 +36,7 @@ traitor-role-codewords =
 traitor-role-uplink-code =
     Set your ringtone to the notes {$code} to lock or unlock your uplink.
     Remember to lock it after, or the stations crew will easily open it too!
+traitor-role-uplink-implant = You have been provided with an uplink implant (for a price). Access it from your hotbar.
```

Comment:
I would adjust the line above to "Use the uplink in your PDA or implant to buy the tools you'll need for this mission."

Reply:
Line 29 now no longer states anything specific about the uplink, leaving that clarification for the last line that the player is more likely to read anyway. It will either state the PDA code, or state that its an implant that you can activate from the hotbar